src/main/java/org/aksw/iguana/cc/metrics/impl/AggregatedExecutionStatistics.java

```diff
@@ -80,7 +79,8 @@ private static Model createAggregatedModel(List<HttpWorker.ExecutionStats> data,
         m.add(queryRes, IPROP.timeOuts, ResourceFactory.createTypedLiteral(timeOuts));
         m.add(queryRes, IPROP.wrongCodes, ResourceFactory.createTypedLiteral(wrongCodes));
         m.add(queryRes, IPROP.unknownException, ResourceFactory.createTypedLiteral(unknownExceptions));
-        m.add(queryRes, IPROP.totalTime, ResourceFactory.createTypedLiteral(toXSDDurationInSeconds(totalTime)));
+        m.add(queryRes, IPROP.totalTime, TimeUtils.createTypedDurationLiteralInSeconds(totalTime));
+        // m.add(queryRes, IPROP.totalTime, TimeUtils.createTypedDurationLiteral(totalTime));
```

Comment:
Add an TODO to commented out code so that we remember what it was all about.

Reply:
I'm probably going to remove the toXSDDurationInSeconds method, because it returned an XSDDuration object which was then immediately converted into a typed literal. I think the method should just return the typed literal itself, like the `createTypedDurationLiteral` method did before.
When we're reverting the change, I will probably keep the `createTypeDurationLiteralInSeconds` method and update it. 

The toXSDDurationInSeconds method can be removed I think.